### PR TITLE
Add Greek language

### PR DESCRIPTION
**Pull request by Twilight0**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [ ] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

Help greek Kodi users use more legit sources, and netflix in this particular case

#### Describe the new behavior

The addon should now have a full UI in greek language.

### Screenshots (if appropriate):

N/A

### CODE REVIEW

**Comment by CastagnaIT** [comment]
> The addon should now have a full UI in greek language.

_should?_

you have tested the changes or not?

File: addon.xml [hunk 1]
```diff
@@ -24,6 +24,9 @@
     <summary lang="en_GB">Netflix</summary>
     <description lang="en_GB">Netflix VOD Services Addon</description>
     <disclaimer lang="en_GB">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
+    <summary lang="en_GB">Νέτφλιξ</summary>
```
[the remark this answers is not in the document]
Comment (Twilight0) [reply addon.xml]: Corrected on next commit.

File: addon.xml [hunk 2]
```diff
@@ -24,6 +24,9 @@
...
+    <description lang="en_GB">Πρόσθετο υπηρεσιών κατά παραγγελία του Νέτφλιξ</description>
```
[the remark this answers is not in the document]
Comment (Twilight0) [reply addon.xml]: Corrected on next commit.

File: addon.xml [hunk 3]
```diff
@@ -24,6 +24,9 @@
...
+    <disclaimer lang="en_GB">Ορισμένα μέρη αυτού του προσθέτου μπορεί να μην είναι νόμιμα στην χώρα που κατοικείτε - παρακαλώ ελέγξτε τους τοπικούς νόμους πριν την εγκατάσταση.</disclaimer>
```
[the remark this answers is not in the document]
Comment (Twilight0) [reply addon.xml]: Corrected on next commit.

File: resources/language/resource.language.el_gr/strings.po [hunk 4]
```diff
@@ -0,0 +1,982 @@
...
+# Addon id: plugin.video.netflix
+# Addon Provider: libdev + jojo + asciidisco + caphm + CastagnaIT
+msgid ""
+msgstr ""
+"Project-Id-Version: plugin.video.netflix\n"
+"Report-Msgid-Bugs-To: https://github.com/CastagnaIT/plugin.video.netflix\n"
+"POT-Creation-Date: 2017-07-24 16:15+0000\n"
+"PO-Revision-Date: 2020-03-09 20:40+0000\n"
+"Last-Translator: Twilight0 <>\n"
+"Language-Team: Greek\n"
```
[the remark this answers is not in the document]
Comment (Twilight0) [reply resources/language/resource.language.el_gr/strings.po]: This line has no effect whatsoever, but If you want the addition, so be it.

File: resources/language/resource.language.el_gr/strings.po [hunk 5]
```diff
@@ -0,0 +1,982 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: el\n"
```
[the remark this answers is not in the document]
Comment (Twilight0) [reply resources/language/resource.language.el_gr/strings.po]: This line has no effect whatsoever, but If you want the addition, so be it.

**Comment by Twilight0** [comment]
You 're right, some lines in addon.xml were missed. 

> > The addon should now have a full UI in greek language.
> 
> _should?_
> 
> you have tested the changes or not?

I don't have a premium netflix account to test, but I intent to. This is a simple translation, so it should work out of the box.

**Comment by CastagnaIT** [comment]
there is no requirement to use a Premium account..

**Comment by CastagnaIT** [comment]
I will leave this translation pending, until the new release version is defined
More changes will be made before the release

**Comment by Twilight0** [comment]
> I will leave this translation pending, until the new release version is defined
> More changes will be made before the release

Fair enough, I 'll make sure to rebase after the changes have been applied.

**Comment by Twilight0** [comment]
Can I now rebase and resolve changes, after that last PR you merged?

**Comment by CastagnaIT** [comment]
the new release is not planned yet
you are free to do as you wish

**Comment by CastagnaIT** [comment]
Currently i do not plan any other changes, you can update it

**Comment by Twilight0** [comment]
Done

**Pull request merged**